AIFS_ROS/hiperlab_rostools/src/AirSimBridge/main.cpp

```diff
@@ -44,12 +44,22 @@
 #include "std_msgs/Bool.h"
 using namespace std;
 
-class ImagePollAgent {
+class SingleImagePollAgent {
  public:
+  SingleImagePollAgent(ImageCaptureBase::ImageType requestedImageType =
+                           ImageCaptureBase::ImageType::DepthVis,
+                       ImageCaptureBase::ImageType requestedImageType2 =
+                           ImageCaptureBase::ImageType::Scene) {
+    imageType = requestedImageType;
+    imageType2 = requestedImageType2;
+  }
+  ImageCaptureBase::ImageType imageType;
+  ImageCaptureBase::ImageType imageType2;
   std::shared_ptr<ros::Subscriber> depthPollTrigger;
   unsigned imageCount = 0;
   std::shared_ptr<Simulation::SimulationObject6DOF> vehicle;
   std::shared_ptr<image_transport::Publisher> pubDepthImage;
+  std::shared_ptr<image_transport::Publisher> pubOtherImage;
```

Comment:
Let us call it RGB image for readability 
The idea is that we provide two sensor options: depth and rgb. User can choose to use either or both. 
Just ctrl+f all "other" and replace with "rgb"